packages/docs/code-snippets/external/example-image-gallery/src/css/style.css

```diff
@@ -0,0 +1,364 @@
+
+@import url(https://weloveiconfonts.com/api/?family=fontawesome);
```

Comment:
@yusefnapora try and keep example sites self contained for much CID-based self-contained portability, and *checks notes* _performance_! yep the old theory that _"they user might have loaded this font already from another site so it'll be faster"_ is no more. https://wicki.io/posts/2020-11-goodbye-google-fonts/